src/QMCDrivers/DMC/DMCDriverInput.cpp

```diff
@@ -38,13 +38,22 @@ void DMCDriverInput::readXML(xmlNodePtr node)
   parameter_set_.add(alpha_, "alpha", "double");
   parameter_set_.add(gamma_, "gamma", "double");
 
-  parameter_set_.put(node);
+  // DMC target walkers is MPI_world scope
+  parameter_set_.add(target_walkers_, "walkers", "int");  
+  parameter_set_.add(target_walkers_, "target_walkers", "int");
```

Comment:
Let us stick to universal `total_walkers` for both VMC and DMC and have the tightest control in the beginning.